ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -581,6 +581,68 @@ module.exports = function OpenlayersMap(
       })
       shapes.push(line)
     },
+    showMultiLineShape: function(locationModel) {
+      var lineObject = locationModel.get('multiline').map(line => {
+        return line.map(coords => {
+          return convertPointCoordinate(coords)
+        })
+      })
+
+      var feature = new Openlayers.Feature({
```

Comment:
Here down is the same in both of these functions - we could break this out into a render vectorShape that takes the type as a param reduce code and prevent additional copypastes.